frontend/src/components/VariantListPage/VariantsTable.tsx

```diff
@@ -672,79 +679,102 @@ const VariantsTable: FC<VariantsTableProps> = ({
       columns: ColumnDef[];
       data: any;
     };
-    style: any;
+    style: React.CSSProperties;
   }) => {
     const rowData = data[dataRowIndex];
-    const rowIndex = dataRowIndex + 1;
+    const isNotIncluded = notIncludedVariants.has(rowData.id);
+
     return (
       <Tr
-        key={rowIndex}
+        key={dataRowIndex}
         sx={{
           display: "flex",
           flexDirection: "row",
           boxSizing: "border-box",
           height: `${ROW_HEIGHT}px`,
-          background: "inital",
+          backgroundColor: isNotIncluded ? "#f0f0f0" : "transparent",
         }}
         style={style}
       >
         {includeCheckboxColumn && (
+          <Td sx={{ height: `${ROW_HEIGHT}px`, alignContent: "center" }}>
+            <Checkbox
```

Comment:
This could be a good place to conditionally set the ability of the checkbox to be toggled, i.e. disallow checking of the box if it is on the not include in calculations list / is grey. As it stands, I can check the box to include it in calculations after checking the box to always remove it.

![Screenshot 2024-10-07 at 15 53 49](https://github.com/user-attachments/assets/d5fcbc16-e372-48e7-9fd9-cd0bccc9fc54)


Similar to above, this is pretty minor, but would be a nice bit of polish.